mne_connectivity/spectral/time.py

```diff
@@ -443,19 +582,33 @@ def _spectral_connectivity(data, method, kernel, foi_idx,
         frequency.
     mt_bandwidth : float | None
         Multitaper time-bandwidth.
+    gc_n_lags : int
+        Number of lags to use for the vector autoregressive model when
+        computing Granger causality.
+    rank : tuple of array
+        Ranks to project the seed and target data to.
     decim : int
         Decimation factor after time-frequency
         decomposition.
     padding : float
         Amount of time to consider as padding at the beginning and end of each
         epoch in seconds.
+    multivariate_con : bool
+        Whether or not multivariate connectivity is to be computed.
 
     Returns
     -------
-    this_conn : list of array
-        List of connectivity estimates corresponding to the metrics in
-        ``method``. Each element is an array of shape (n_pairs, n_freqs) or
-        (n_pairs, n_fbands) if ``faverage`` is `True`.
+    scores : dict
+        Dictionary containing the connectivity estimates corresponding to the
+        metrics in ``method``. Each element is an array of shape (n_pairs,
+        n_freqs) or (n_pairs, n_fbands) if ``faverage`` is `True`.
+
+    patterns : dict
+        Dictionary containing the connectivity patterns corresponding to the
+        metrics in ``method``, if multivariate methods are called, else an
+        empty dictionary. Each element is an array of shape (n_pairs,
+        n_channels, n_freqs) or (n_pairs, n_channels, n_fbands) if ``faverage``
+        is `True`.
```

Comment:
Sorry I'm not familiar. Can you elaborate what each element represents here?

From https://mne.tools/dev/generated/mne.decoding.SSD.html, I get `The patterns for reconstructing the signal from the filtered data.`, but I presume that's not exactly what it is here? 

Just want to make sure it's crystal clear :p.

Reply:
So it's very similar, but instead of reconstructing the signal we're reconstructing the largest connectivity component of the signal. I have updated to try and make it clearer:
https://github.com/tsbinns/mne-connectivity/blob/894c42e59cef9f97aa07a7a4f71cef756512c50c/mne_connectivity/spectral/time.py#L613-L619